src/test/java/picard/sam/SplitSamByNumberOfReadsTest.java

```diff
@@ -251,4 +251,21 @@ private void compareInputWithOutputs(final SamReader reader1, final SamReader re
             Assert.assertEquals(rec.getReadName(), inputRec.getReadName());
         }
     }
+
+    @Test
+    public void testOutPrefixWithZeros() throws IOException {
+        final String tmpDir = IOUtil.getDefaultTmpDir().getAbsolutePath();
```

Comment:
When writing tests I don't think it's necessary to use the `IOUtil` code for temp directories. Instead I would use `Files.createTempFile()` (or `File.createTempFile()`) below. Also, `getAbsolutePath()` is unnecessary unless you're printing out the path and you want to avoid relatives paths.